daily-challenges/2024-11/29/lcaloi.29.spec.ts

```
@@ -0,0 +1,25 @@
+import { expect, test } from '@playwright/test';
+
+test('solution for day 29/11/24', async ({ page }) => {
+    await page.goto('https://material.playwrightvn.com/');
+    await expect(page.getByText('Tài liệu học automation test')).toBeVisible();
+
+    await page.getByRole('link', { name: 'Game 04: Trang trí cây thông Noel' }).click();
+    await expect(page.getByText('Trang Trí Cây Thông Noel')).toBeVisible();
+
+    let listStyles = await page.locator('ul[style]>li').all();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'listStyles' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)